keyboards/e6v2/le/keymaps/johu/readme.md

```diff
@@ -0,0 +1,38 @@
+# johu's E6-V2 Last Edition (LE) HHKB
+Make example for this keyboard (after setting up your build environment):
+```bash
+$ make e6v2/le:johu
+```
+
+Or to make and flash:
+```bash
+$ make e6v2/le:johu:dfu
+```
+
+See [build environment setup] then the [make instructions] for more information.
```

Comment:
Please change this line to:

`See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).`

and then delete Lines 35 and 36.

Reply:
I will add the the last sentence, but I prefer the links at the bottom as it the markdown is way more readable in source.

Reply:
This information should also be in the keyboard's readme, not just in a keymap.

Reply:
> This information should also be in the keyboard's readme, not just in a keymap readme.

Done